docs/content/en/docs/reference/clusterspec/optional/irsa.md

```diff
@@ -83,41 +110,65 @@ Set the remaining fields in [cluster spec]({{< relref "../vsphere/" >}}) as requ
 
-1. Check out [this commit](https://github.com/aws/amazon-eks-pod-identity-webhook/commit/a65cc3d9c61cf6fc43f0f985818c474e0867d786) of the amazon-eks-pod-identity-webhook.
+1. Clone [amazon-eks-pod-identity-webhook](https://github.com/aws/amazon-eks-pod-identity-webhook) if not done already.
+
+2. Set the $KUBECONFIG env var to the path of the EKS Anywhere cluster.
+
+3. Update `amazon-eks-pod-identity-webhook/deploy/auth.yaml` with `OIDC_IAM_ROLE` and other annotations as mentioned in sample below.
+    ```yaml
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: my-serviceaccount
+      namespace: default
+      annotations:
+        # set this with value of OIDC_IAM_ROLE      
+        eks.amazonaws.com/role-arn: "arn:aws:iam::111122223333:role/s3-reader"
+        # optional: Defaults to "sts.amazonaws.com" if not set
+        eks.amazonaws.com/audience: "sts.amazonaws.com"
+        # optional: When set to "true", adds AWS_STS_REGIONAL_ENDPOINTS env var
+        #   to containers
+        eks.amazonaws.com/sts-regional-endpoints: "true"
+        # optional: Defaults to 86400 for expirationSeconds if not set
+        #   Note: This value can be overwritten if specified in the pod 
+        #         annotation as shown in the next step.
+        eks.amazonaws.com/token-expiration: "86400"
+    ```
 
-1. Set the $KUBECONFIG env var to the path of the EKS Anywhere cluster.
-1. Run the following command:
+4. Run the following command:
 
     ```bash
-    make cluster-up IMAGE=amazon/amazon-eks-pod-identity-webhook:a65cc3d
+    make cluster-up IMAGE=amazon/amazon-eks-pod-identity-webhook:latest
     ```
+5. You can validate IRSA by using test steps mentioned [here](https://anywhere.eks.amazonaws.com/docs/workshops/packages/adot/adot_amp_amg/#irsa-set-up-test). Ensure awscli pod is deployed in same namespace of ServiceAccount pod-identity-webhook.
 
 
-#### Configure the trust relationship for the OIDC provider's IAM Role
+### Configure the trust relationship for the OIDC provider's IAM Role
 
 In order to grant certain service accounts access to the desired AWS resources, edit the trust relationship for the OIDC provider's IAM Role (`OIDC_IAM_ROLE`) created in the first section, and add in the desired service accounts.
 1. Choose the role in the console to open it for editing.
-1. Choose the **Trust relationships** tab, and then choose **Edit trust relationship**.
-1. Find the line that looks similar to the following:
+2. Choose the **Trust relationships** tab, and then choose **Edit trust relationship**.
```

Comment:
Change steps 2, 3, 4, 5, and 6 to "1."

Reply:
Thanks for your observations. I've updated and committed the changes.